lib/pdf_fill/extras_generator.rb

```diff
@@ -24,15 +24,32 @@ def text?
       @generate_blocks.size.positive?
     end
 
+    def generate_header(pdf)
+      pdf.repeat(:all) do
+        pdf.bounding_box([pdf.bounds.left, pdf.bounds.top], width: pdf.bounds.width) do
+          pdf.text('Vets.gov Submission', align: :right)
+        end
+      end
+    end
+
     def generate
       folder = 'tmp/pdfs'
       FileUtils.mkdir_p(folder)
       file_path = "#{folder}/extras_#{Time.zone.now}.pdf"
```

Comment:
Could this use a Tempfile instead of generating a directory? If not is this dir unique? What happens if two PDFs are stamped at the same time? (`SecureRandom.uuid` may be better for a unique file name)